### PR TITLE
CSV upload skeleton

### PR DESCRIPTION
**Pull request by weronica**
User can upload a CSV file. Contents of CSV file are printed to terminal.

- Added `bulk` blueprint, which will contain all steps of bulk resource upload (CSV file upload, multi-stage review process, final resource saving).
- Added route and form to handle CSV file upload.

r @armant

### CODE REVIEW

**Comment by armant** [comment]
r @dzhang55

File: app/bulk/forms.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from flask.ext.wtf import Form
+from wtforms.fields import (
```
Comment (dzhang55) [inline app/bulk/forms.py]: should these be alphabetical?

Comment (weronica) [inline app/bulk/forms.py]: Done.

File: app/bulk/views.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from flask import redirect, render_template, url_for
+from flask.ext.login import (
```
Comment (armant) [inline app/bulk/views.py]: make this import statement a one-liner?

File: app/bulk/views.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    login_required
+)
+from . import bulk
+from forms import (
```
Comment (armant) [inline app/bulk/views.py]: nit: make this import statement a one-liner?

File: app/bulk/views.py (hunk 2 above)
Comment (weronica) [inline app/bulk/views.py]: Done

**Comment by armant** [comment]
LGTM

**Pull request merged**